### PR TITLE
OSDOCS-9119: Query Loki manually troubleshooting

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OSDOCS-9119
Link to docs preview:

https://74219--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/troubleshooting-network-observability.html#troubleshooting-query-loki-manually_network-observability-troubleshooting
QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/troubleshooting-network-observability-query-loki-manually.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting-query-loki-manually_{context}"]
+= Running custom queries to Loki
+
+For troubleshooting, you might be interested in running your own custom queries to Loki. There are two examples of ways to do this, which you can adapt according to your needs by replacing the <api_token> with your own.
```
Comment (jotak) [inline modules/troubleshooting-network-observability-query-loki-manually.adoc:9]: Also these examples assume both netobserv and loki are deployed in namespace `netobserv`, and that the lokistack was named `loki`. If the users have something different they need to adapt the examples as well (the `-n netobserv` and/or the loki gateway URL)

Comment (skrthomas) [reply modules/troubleshooting-network-observability-query-loki-manually.adoc:9]: Ok gotcha, we have a similar note in the Installation section. I created a note like that here with these details. Thanks for the info. I agree its probably good to point these out specifically.

**Review by memodi — APPROVED** [review]
/label qe-approved

**Review by jotak — APPROVED** [review]

**Review by tmalove — APPROVED** [review]

**Comment by skrthomas** [comment]
Thanks @tmalove ! Addressed both :)

**Pull request merged**